app/forms/hydra_editor/form.rb

```diff
@@ -1,7 +1,6 @@
 module HydraEditor
   module Form
     extend ActiveSupport::Autoload
```

Comment:
I think this line could be removed as well.  Do you mind trying that?

Reply:
I tested this with my hyrax 4 setup and the error didn't reappear. I tried to test it with my hyrax 3 setup but it didn't work.

Reply:
Does removing just the `autoload :Permissions` line break your Hyrax 3?

Reply:
Since I forked it off hydra-editor when I tried to use that branch I get this with hyrax 3 during the bundle install. I wasn't able to test it on this version.

27.11 Could not find compatible versions
27.11 
27.11 Because hydra-editor >= 5.0.4, < 6.A could not be found in
27.11 https://github.com/wvulibraries/hydra-editor (at main@d2d687f)

Reply:
Doing some digging I see Hyrax 4 wants hydra-editor (~> 6.0) while  hyrax 3 is trying to install hydra-editor (5.0.5). I forked main and set Hyrax 4 to my fork.

Reply:
I brought this PR up on the Samvera tech call and the consensus was that this looks good and that removing the extend should be safe to do as well.
```suggestion
```
What do you think of adding this to your PR?  I'm happy to approve this either way.

Reply:
I'm sure that's fine I can add it and test it tomorrow.